analyzer/src/funTest/kotlin/integration/DrupalIntegrationFunTest.kt

```diff
@@ -43,10 +43,10 @@ class DrupalIntegrationFunTest : AbstractIntegrationSpec() {
         binaryArtifact = RemoteArtifact.EMPTY,
         sourceArtifact = RemoteArtifact.EMPTY,
         vcs = VcsInfo(
-            VcsType.GIT,
-            "https://github.com/drupal/drupal.git",
-            "4a765491d80d1bcb11e542ffafccf10aef05b853",
-            ""
+            type = VcsType.GIT,
+            url = "https://github.com/drupal/drupal.git",
+            revision = "4a765491d80d1bcb11e542ffafccf10aef05b853",
+            isResolvedRevision = true
```

Comment:
As discussed in the developer meeting, `isVerifiedRevision` or `isConfirmedRevision` would probably fit the semantics better.

Reply:
I have renamed it to `isVerifiedRevision` and also removed the last commit because you mentioned in the meeting today that NuGet takes the commit SHA1 from metadata so it would be wrong to set the flag there.

Reply:
As discussed today I have now completely removed the `isVerifiedRevision` property so that we can address this topic later in a more sophisticated way.